docs_src/run_camera_simulator.md

```diff
@@ -1,32 +1,40 @@
-Before running the below (e.g. starting the camera-simulator.sh) ensure you have downloaded video file(s) to the `sample-media` directory. Execute the commands 
-```bash
-cd benchmark-scripts; sudo ./download_sample_videos.sh; cd ..;
-``` 
-These commands are provided as an option to download a sample video(s) and RTSP stream with the camera-simulator.  You can also specify the desired resolution and framerate e.g.
-```bash
-cd benchmark-scripts; sudo ./download_sample_videos.sh 1920 1080 15; cd ..;
-```
-for 1080p@15fps. Note that only AVC encoded files are supported.
+If you do not have a camera device plugged into the system, run the camera simulator to view the pipeline analytic results based on a sample video file to mimic real time camera video. You can also use the camera simulator to infinitely loop through a video file for consistent benchmarking. For example, if you want to validate whether the performance is the same for 6 hours, 12 hours, and 24 hours, looping the same video should produce the same results regardless of the duration.
 
-Once video files are copied/downloaded to the sample-media folder, start the camera simulator from automated-self-checkout/ directory with:
-```bash
-make run-camera-simulator
-``` 
+Do the following to run the cameral simulator:
 
-!!!Note Please wait for few seconds, then use below command to check if camera-simulator containers are running.
+1. Download the video files to the **sample-media** directory:
+    ```bash
+    cd benchmark-scripts;
+    sudo ./download_sample_videos.sh;
+    cd ..;
+    ```
```

Comment:
remove the leading spaces as the md not working if there is extra leading spaces